src/rust/engine/Cargo.lock

```diff
@@ -616,7 +616,7 @@ name = "double-checked-cell-async"
 version = "2.0.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "futures-util 0.3.4 (registry+https://github.com/rust-lang/crates.io-index)",
+ "futures-util 0.3.5 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
@tdyas fyi the lock file wasn't updated, I think. Running `./pants` should be enough to trigger the update.

Reply:
I noticed that. I have two approved PRs pending merges later that include that fix. So whatever fixes it first!